c2corg_ui/format/ltag.py

```diff
@@ -0,0 +1,64 @@
+"""
+LTag Extension for Python-Markdown
+====================================
+
+See https://regex101.com/r/g2Zjqn/2/ for regex tests
+
+"""
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from markdown.extensions import Extension
+from markdown.blockprocessors import BlockProcessor
+from markdown.util import etree
+import re
+
+
+class LTagProcessor(BlockProcessor):
+    """ Process Tables. """
+
+    RE_LTAG_BLOCK = re.compile(r'^(?:L#(?:[^|\n]*(?:\|[^|\n]*)+)\n?)+$')
+
+    def __init__(self, parser):
+        super(LTagProcessor, self).__init__(parser)
+
+    def test(self, parent, block):
+        is_table = self.RE_LTAG_BLOCK.search(block) is not None
+
+        return is_table
+
+    def run(self, parent, blocks):
+
+        block = blocks.pop(0)
+        row_count = 1
+
+        if not self.test(parent, block):
+            return
+
+        rows = block.split('\n')
+        table = etree.SubElement(parent, 'table')
+        table.set('class', 'ltag')
+        tbody = etree.SubElement(table, 'tbody')
+        for row in rows:
+            tr = etree.SubElement(tbody, 'tr')
+            cols = row.split('|')
+            for col in cols:
+                td = etree.SubElement(tr, 'td')
+                td.text = col.strip().replace('L#', "L" + str(row_count))
```

Comment:
In v5, the pitch code was translated differently depending on the interface language.
For instance "L10" in french but "P10" in english (P = pitch). This could be done by using the following HTML:
```
<span translate>L</span>10
```

Please note there are also "Rtags" (R#) for rappels :P 
https://www.camptocamp.org/articles/305462/fr/aide-topoguide-l-balise-de-description-des-longueurs#description-des-rappels

Not clue if this is needed but I remember that the pitch code was bold in v5. Then perhaps adding a "styling" element would be useful:
```
<span class="pitch"><span translate>L</span>10</span>
```
Up to you :) Those changes can also be done in a later PR.